data/json/items/ammo.json

```diff
@@ -946,6 +946,7 @@
         "category" : "chems",
         "price" : 3500,
         "name" : "sprayable fungicide",
+        "name_plural" : "sprayable fungicide",
```

Comment:
Why do you need to translate these phrases at all if they are not shown to anyone?